src/components/home/sections/RelatedCards.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { CardDeck, Card } from "react-bootstrap";
+import logo1 from "../related-cards-logos/Logo1.png";
```

Comment:
let's name these files lowercase and starting with 0 index, which is more conventional

`logo0.png, logo1.png`, etc.

Reply:
i think you can also rename the directory `/home/logos/`